lib/Frontend/Frontend.cpp

```diff
@@ -26,6 +26,7 @@
 #include "swift/Parse/DelayedParsingCallbacks.h"
 #include "swift/Parse/Lexer.h"
 #include "swift/SIL/SILModule.h"
+#include "swift/Basic/Statistic.h"
```

Comment:
Nitpick: please sort this by library name within the top-level group. See the [LLVM coding standards](http://llvm.org/docs/CodingStandards.html#include-style) for more info.

Reply:
I had thought I had used git-clang-format and that it would do the sort, but I've fixed it manually and will git-clang-format again before pushing. Tx for catching this.